src/region_growing/label_connected_comp.py

```diff
@@ -78,17 +96,43 @@ def fill_components(self, seed_point_label, threshold=0.1):
                 continue
             # number of point in teh cluster that are labelled as seed point
             seed_count = np.count_nonzero(
-                self.las_label[mask_indices[cc_mask]] == seed_point_label)
+                self.las_label[mask_indices[cc_mask]] == self.seed_point_label)
             # at least X% of the cluster should be seed points
             if (float(seed_count) / cc_size) > threshold:
                 label_mask[mask_indices[cc_mask]] = True
 
-        # Add label to the regions
+        # Add label to the regions TODO kan dit niet makkelijker??
```

Comment:
Best veel dingen om post_seed_count te berekenen

Reply:
Wat bedoel je? Dat is maar 1 regel hieronder. Kan ook weg, want was puur voor een debug printout :)